source/renderer/app/components/static/About.scss

```diff
@@ -1,11 +1,39 @@
 .container {
-  background-color: var(--theme-about-window-background-color);
   display: flex;
   flex-direction: column;
   height: 100%;
-  padding: 30px 30px 20px;
+  padding: 50px 80px 50px;
```

Comment:
@DeeJayElly please change to: `padding: 50px 80px;`

Reply:
@nikolaglumac Done